### ISSUE TITLE
Code Syntax on File activation not checked, when Roslynator is activated.

### ISSUE THREAD

**Issue by mz-zmsoft** [OmniSharp/omnisharp-roslyn#1674]
**Product and Version Used**: 
VSCODE 1.41.1 on OSX
josefpihrt-vscode.roslynator 2.3.0
ms-vscode.csharp 1.21.9

**Steps to Reproduce**:
- Open any C# based project
- Create a new folder / namespace in this project called "Demo"
- Create a new Class within Demo Folder "Class1".
- Open Class1.cs  and add following line:
```csharp
  private Class2 demo = null;
```
- Use Intellisense for Class2 File Creation ("Generate class Class2 in separate File")
- Open the new created File "Class2.cs" and save it!
- Change back to File "Class1.cs".

![image](https://user-images.githubusercontent.com/55440373/71640216-ec014480-2c85-11ea-84b1-58616fceade3.png)

- ```Class2``` keyword is still red marked. Even when saving "Class1.cs" the red error remains.
- Only way to resolve this issue is to edit the file on any position (insert space or whatever). It seems like a "file input" event will force a re-check of the syntax. The file check succeeds now.

![image](https://user-images.githubusercontent.com/55440373/71640235-83669780-2c86-11ea-98b8-388ee6fbce1d.png)

**Actual Behavior**:
Syntax check isn't triggered, when a source file is focused.

**Expected Behavior**:
Syntax check is triggered immediately, when a source file is focused. This behavior is woking, when the Extension 'josefpihrt-vscode.roslynator' is deactivated.

**Comment by filipw** [comment]
OmniSharp only reacts to the events sent by the editor. If the editor doesn't send the events, or sends them in a wrong order, OmniSharp has no way of knowing what is the correct behavior. Based on what you are explaining here, please open this issue in the Roslynator repo?
